experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -321,22 +321,15 @@ export class FetchInstrumentation extends InstrumentationBase<Promise<Response>>
 
         function endSpanOnSuccess(span: api.Span, response: Response) {
           plugin._applyAttributesAfterFetch(span, options, response);
-          const spanResponse = {
```

Comment:
weird that the unused variable didn't cause compilation to fail

Reply:
The assignment to the property of the object caused eslint to think it was used.